controllers/suite_test.go

```diff
@@ -41,10 +41,7 @@ var testEnv *envtest.Environment
 
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
-
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
+	RunSpecs(t, "Controller Suite")
```

Comment:
How did you know that you need to use `RunSpecs`?

Did you read some documentation somewhere or just searched for the replacement of `RunSpecsWithDefaultAndCustomReporters` method in the "[Files changed](https://github.com/kubernetes-sigs/controller-runtime/pull/1977)" tab of the PR which removed the `pkg/envtest/printer` package?

Reply:
https://github.com/kubernetes-sigs/controller-runtime/pull/1977/files#diff-ebf5180c1611182be731c4d10224db0cd70eda680fd8c9a42caf437760739fefL42

I followed how they updated their testing suite.

Reply:
Thanks!